addons/mail/models/models.py

```diff
@@ -198,6 +198,12 @@ def _get_default_activity_view(self):
     # GATEWAY: NOTIFICATION
     # ------------------------------------------------------------
 
+    @api.model
+    def _get_mail_message_subtypes(self):
```

Comment:
Maybe ``def _mail_get_message_subtypes`` so that it is prefixed by ``_mail``, indicating this is used in the mail mixin